types/chart.js/chart.js-tests.ts

```diff
@@ -241,3 +241,31 @@ const chartWithScriptedOptions = new Chart(new CanvasRenderingContext2D(), {
         }],
     }
 });
+
+const lineChartOptions: ChartOptions = {
+    responsive: true,
+    scales: {
+        xAxes: [{
+            id: 'x-axis-0',
+            position: 'bottom'
+        }],
+        yAxes: [
+            {
+                id: 'y-axis-0',
+                display: 'auto' as any,
```

Comment:
with the new ts definition the cast `as any` should not be needed

```suggestion
                display: 'auto',
```